src/VMware/custom/New-AzVMwarePrivateCloud.ps1

```diff
@@ -185,12 +190,35 @@ function New-AzVMwarePrivateCloud {
     )
     
     process {
+
+        if(!$PSBoundParameters.ContainsKey('AcceptEULA')){
+            try {
+                $legalTermPath = Join-Path $PSScriptRoot -ChildPath "LegalTerm.txt"
+                $legalTerm = (Get-Content -Path $legalTermPath) -join "`r`n"
+
+                $confirmation = Read-Host $legalTerm"`n[Y] Yes  [N] No  (default is `"N`")"
+                switch ($confirmation) {
+                    'Y' {
+                        Break
+                    }
+    
+                    Default {
+                        Return
+                    }
+                }
+            } catch {
+                throw
```

Comment:
What error message will be shown up? Could you add an example on your PR description?

Reply:
wrap raw exception

Reply:
https://docs.microsoft.com/en-us/powershell/scripting/learn/deep-dives/everything-about-exceptions?view=powershell-7.1#re-throwing-a-new-exception

Please follow this example and wrap system exception